webapp/channels/src/actions/telemetry_actions.jsx

```diff
@@ -204,25 +204,39 @@ export function trackPluginInitialization(plugins) {
     });
 }
 
+function getTopThreeSelectors(selectors) {
+    const topThreeSelectors = [];
+    for (const selector of selectors) {
+        if (topThreeSelectors.length === 3) {
+            break;
+        }
+        if (selector.calls > 5) {
+            topThreeSelectors.push(selector);
+        }
+    }
+    return topThreeSelectors;
+}
+
 export function trackSelectorMetrics() {
     setTimeout(() => {
         if (!shouldTrackPerformance()) {
             return;
         }
 
         const selectors = getSortedTrackedSelectors();
+        const topThreeSelectors = getTopThreeSelectors(selectors);
```

Comment:
Do you think it's worth keeping `getSortedTrackedSelectors` as-is, or could we just move `getTopThreeSelectors` to be part of `getSortedTrackedSelectors`? It seems like that method isn't particularly helpful currently because of anywhere that makes a one-off selector

Reply:
I do like looking at `window. getSortedTrackedSelectors()` locally so I'd like to keep it as is unless there is a better way to track selectors locally?

Reply:
Glad to know someone's using that since I think we left that on the window specifically for debugging. 😀 

Would it perhaps make that easier to use if `getSortedTrackedSelectors` removes selectors with fewer than 5 calls from it? 0/5 still though

Reply:
Yeah it's very useful, you can reset the tracker too which is nice. I'd like to leave it as is right now